src/libraries/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/Atom10FeedFormatter.cs

```diff
@@ -595,8 +595,6 @@ private static TextSyndicationContent ReadTextContentFromHelper(XmlReader reader
                 }
             }
             reader.MoveToElement();
-            string localName = reader.LocalName;
-            string nameSpace = reader.NamespaceURI;
```

Comment:
Do either LocalName or NamespaceURI have side-effects, e.g. advancing the reader, throwing exceptions if something couldn't be parsed, etc.?

Reply:
> advancing the reader

`LocalName`/`NamespaceURI` makes the reader query information of the current **node** so there is no streaming.

> throwing exceptions if something couldn't be parsed

`LocalName` and `NamespaceURI` are documented to not throw XML (parsing) exceptions.